lnd_test.go

```diff
@@ -1464,6 +1468,150 @@ func testMultiHopPayments(net *networkHarness, t *harnessTest) {
 	}
 }
 
+func testMaxFee(net *networkHarness, t *harnessTest) {
```

Comment:
Missing a `godoc` style comment describing what functionality is under test by this function.